src/services/videoconference/logic/server.js

```diff
@@ -0,0 +1,20 @@
+const bbb = require('bbb-promise');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'bbb-promise'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)